themes/tdr/login/info.ftl

```diff
@@ -1,5 +1,6 @@
 <#import "template.ftl" as layout>
 <#assign backToApplicationText = kcSanitize(msg("backToApplication"))>
+<#assign cnotinueToApplicationText = kcSanitize(msg("continueToApplication"))>
```

Comment:
Spelling of variable. Should be: `continueToApplicationText`

Reply:
Good catch. I have updated that, thanks.